### PR TITLE
FINE-1099642259

### PR DESCRIPTION
**Pull request by texasalgonaut**

### CODE REVIEW

**Review by texasalgonaut — COMMENTED** [review]
icon

**Review by texasalgonaut — COMMENTED** [review]
icon

**Comment by texasalgonaut** [comment]
Please replace the icons I previously submitted with these new icons. The old icons did not have a transparent background.

**Comment by ertekinno** [comment]
Hello, thanks for sending icons. Unfortunately, this commit is not suitable to the repository.

- First, you should create a folder. There should be a folder named with `unit_name-asset_id`, e.g. "USDC-31566704" without adding any space. And icon files should be in this folder.
- Icon names should be `icon.png` and `icon.svg`

Please update your commit according to this feedback. You can also view all the requirements from [README.md](https://github.com/tinymanorg/asa-list/tree/main#readme) before creating a pull request.

File: FINE-1099642259 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (texasalgonaut) [inline FINE-1099642259:1]: icon

Comment (texasalgonaut) [inline FINE-1099642259:1]: icon

**Review by texasalgonaut — COMMENTED** [review]
These icons have a transparent background and should replace the originals.

**Pull request closed without merging**